etc/ReplayOfflineConfiguration.py

```diff
@@ -338,7 +337,7 @@
                  sizePerEvent=1700,
                  versionOverride=expressVersionOverride,
                  diskNode="T0_CH_CERN_Disk")
- 
+
 addExpressConfig(tier0Config, "ALCALUMIPIXELSEXPRESS",
```

Comment:
The `ALCALUMIPIXELSEXPRESS` has been replaced by `ALCALumiPixelCountsExpress` (which you coirrectly added below), does it make sense to keep this one?

Reply:
Sorry my bad again I missed the first comment! Then it's fine for me.